wazo_confd/plugins/user/schema.py

```diff
@@ -229,3 +243,37 @@ def on_bind_field(self, field_name, field_obj):
         ]
         if field_name in nullable_fields:
             field_obj.allow_none = True
+
+
+class UserListItemSchema(UserSchemaNullable):
+    auth = Nested('WazoAuthUserSchema')
+    lines = Nested('LineSchema', many=True)
+    incalls = Nested('UserIncallSchema', many=True)
+    groups = Nested('UserGroupSchema', many=True)
+    switchboards = Nested('UserSwitchboardSchema', many=True)
+
+
+class UserIncallSchema(BaseSchema):
+    id = fields.Integer(dump_only=True)
```

Comment:
Most fields from the `incall` schema should be available here.
`preprocess_subroutine`, `greeting_sound`, `caller_id_mode`, `caller_id_name`, `description`